core/src/main/java/org/elasticsearch/search/sort/SortBuilders.java

```diff
@@ -57,14 +57,6 @@ public static ScriptSortBuilder scriptSort(Script script, ScriptSortType type) {
     }
 
     /**
-     * Constructs a new nested sort builder.
-     *
-     */
-    public static NestedSortBuilder nestedSort(String path) {
```

Comment:
I only removed this because it might be misleading to have such a helper in SortBuilders since NestedSortBuilder isn't a top level SortBuilder. Happy to revert this if other think it is helpfull here.